packages/esm-patient-registration-app/src/patient-registration/field/name/name-field.component.tsx

```diff
@@ -19,17 +20,25 @@ function checkNumber(value: string) {
 }
 
 export const NameField = () => {
+  const { t } = useTranslation();
+  const { setCapturePhotoProps, currentPhoto, setFieldValue } = useContext(PatientRegistrationContext);
   const {
     fieldConfigurations: {
-      name: { displayCapturePhoto },
+      name: {
+        displayCapturePhoto,
+        allowUnidentifiedPatients,
+        defaultUnknownGivenName,
+        defaultUnknownFamilyName,
+        displayMiddleName,
+      },
     },
   } = useConfig() as RegistrationConfig;
-  const { t } = useTranslation();
-  const { setCapturePhotoProps, currentPhoto, setFieldValue } = useContext(PatientRegistrationContext);
-  const { fieldConfigurations } = useConfig();
-  const fieldConfigs = fieldConfigurations?.name;
-  const [{ value: unidentified }] = useField('unidentifiedPatient');
-  const nameKnown = !unidentified;
```

Comment:
Don't we need some, if not all of these?

Reply:
I actually re-did the imports from the useConfig.

Reply:
Ah , I see! What about the `useTranslation` hook?

Reply:
It's shifted to the top of the component.